src/VSIX/VSIX.csproj

```diff
@@ -217,19 +217,19 @@
     </Reference>
     <!-- Note the extra project reference to Npgsql which makes msbuild compile it, but which has Private=true -->
     <Reference Include="Npgsql">
-      <HintPath>..\Npgsql\bin\$(Configuration)\net452\Npgsql.dll</HintPath>
+      <HintPath>..\Npgsql\bin\$(Configuration)\net461\Npgsql.dll</HintPath>
```

Comment:
Line 52 of this file has `<TargetFrameworkVersion>v4.6</TargetFrameworkVersion>` which seems to cause the break on AppVeyor.

Reply:
Yeah, thanks... Am probably going to have several iterations until I get this to build on Appveyor (my machine is on Linux).